crowbar_framework/app/models/neutron_service.rb

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+require_relative '../helpers/barclamp/neutron_helper.rb'
```

Comment:
I have commented that validation stuff should be part of the model and not the helper.